strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/storage/repository/aws/AwsObjectFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.carlspring.strongbox.storage.repository.aws;
+
+import org.carlspring.strongbox.storage.repository.CustomConfiguration;
+
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.annotation.XmlElementDecl;
+import javax.xml.bind.annotation.XmlRegistry;
+import javax.xml.namespace.QName;
+
+/**
+ * @author carlspring
+ */
+@XmlRegistry
+public class AwsObjectFactory
+{
+
+
+    @XmlElementDecl(name = "aws-configuration", scope = AwsConfiguration.class)
```

Comment:
Add `substitutionHeadName` and `substitutionHeadNamespace` pointing to the `@XmlElementDecl` from the other `ObjectFactory`.